client/tslint.json

```diff
@@ -7,10 +7,12 @@
     "max-line-length": [true, 140],
     "no-floating-promises": false,
     "no-unused-variable": false, // Memory issues
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
+    "member-ordering": [true, {
+      "order": [
+        "public-before-private",
+        "static-before-instance",
+        "variables-before-functions"
+        ]}
```

Comment:
A similar syntax by the tslint docs should be:

```
"public-static-field",
"public-instance-field",
"public-constructor",
"private-static-field",
"private-instance-field",
"private-constructor",
"public-instance-method",
"protected-instance-method",
"private-instance-method"
```

Reply:
I can't reproduce the warnings you get. my ide seems not to show them.
But you are right, according to the docs `"public-before-private"` and the others are not allowed, yet they still seem to work.

Should I change it to your proposal or to something of the examples?

Reply:
Try with a terminal and `npm run travis -- lint`.

Yes, I've not tried this proposal but it should be close enough :crossed_fingers: